static/docs/reference/generated/kubectl/kubectl-commands.html

```diff
@@ -1503,7 +1503,7 @@ <h2 id="-em-secret-docker-registry-em-"><em>secret docker-registry</em></h2>
   nodes to pull images on your behalf, they must have the credentials.  You can provide this information
   by creating a dockercfg secret and attaching it to your service account.</p>
 <h3 id="usage">Usage</h3>
-<p><code>$ kubectl create docker-registry NAME --docker-username=user --docker-password=password --docker-email=email [--docker-server=string] [--from-file=[key=]source] [--dry-run=server|client|none]</code></p>
+<p><code>$ kubectl create secret docker-registry NAME --docker-username=user --docker-password=password --docker-email=email [--docker-server=string] [--from-file=[key=]source] [--dry-run=server|client|none]</code></p>
```

Comment:
A few problems with this, 
- According to [this commit] (https://github.com/moby/moby/commit/18c9b6c6455f116ae59cde8544413b3d7d294a5e), dockercfg has been migrated to config.json. Even the kubectl create secret docker-registry --help command invocation shows that we currently use the config.json format.

Reply:
Secondly, we need to cover both the cases - when a dockercfg/config.json is present & when it is not. IMO, it would add value.

Reply:
Thirdly, if we're adding both cases it would be worthwhile to note that the command to create a secret in the former case would be:

kubectl create secret docker-registry my-secret --from-file=.dockerconfigjson=path/to/.docker/config.json